### PR TITLE
Add package : libandroid-wordexp

### PR DESCRIPTION
**Pull request by suhan-paradkar**
Closes termux/libandroid-support#8

### CODE REVIEW

File: packages/libandroid-wordexp/wordexp.c [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+//	sigprocmask(SIG_SETMASK, &set, &all_set);
+	if (pid < 0) {
+		close(p[0]);
+		close(p[1]);
+		goto nospace;
+	}
+	if (!pid) {
+		if (p[1] == 1) fcntl(1, F_SETFD, 0);
+		else dup2(p[1], 1);
+		execl("@TERMUX_PREFIX@/bin/sh", "sh", "-c",
```
Comment (xeffyr) [inline packages/libandroid-wordexp/wordexp.c:121]: `@TERMUX_PREFIX@` is being expanded **only for patch files**.

**Review by xeffyr — CHANGES_REQUESTED** [review]

File: packages/libandroid-wordexp/wordexp.c (hunk 1 above)
Comment (suhan-paradkar) [reply packages/libandroid-wordexp/wordexp.c]: Corrected it...

File: packages/proxmark3/build.sh [hunk 2]
```diff
@@ -10,11 +10,12 @@ TERMUX_PKG_BUILD_IN_SRC="true"
...
+	export LDLIBS="-L${TERMUX_PREFIX}/lib"
+	export INCLUDES="-I${TERMUX_PREFIX}/include"
+	TERMUX_PKG_EXTRA_MAKE_ARGS="client CC=$CC CXX=$CXX LD=$CXX cpu_arch=generic SKIPREVENGTEST=1 SKIPQT=1 SKIPBT=1 SKIPPTHREAD=1 PLATFORM=PM3GENERIC"
 }
 
 termux_step_make_install() {
-        install -Dm700 "$TERMUX_PKG_BUILDDIR"/client/proxmark3 \
-                "$TERMUX_PREFIX"/bin/proxmark3
+	install -Dm700 "$TERMUX_PKG_BUILDDIR"/client/proxmark3 \
+		"$TERMUX_PREFIX"/bin/proxmark3
```
Comment (xeffyr) [inline packages/proxmark3/build.sh:20]: Why this change included with wordexp commit?

**Review by xeffyr — CHANGES_REQUESTED** [review]
libandroid-wordexp seems to be ok now. Though there are still questions due to changes unrelated to wordexp.

File: packages/proxmark3/build.sh (hunk 2 above)
Comment (suhan-paradkar) [reply packages/proxmark3/build.sh]: Oh... rebase problems....

**Comment by suhan-paradkar** [comment]
@xeffyr Its done...

**Review by xeffyr — APPROVED** [review]
